ibm/resource_ibm_dns_reverse_record_test.go

```diff
@@ -0,0 +1,88 @@
+package ibm
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/softlayer/softlayer-go/datatypes"
+	"github.com/softlayer/softlayer-go/services"
+)
+
+func TestAccIBMDNSReverseRecord_Basic(t *testing.T) {
+	var dns_domain datatypes.Dns_Domain
+	var dns_domain_record datatypes.Dns_Domain_ResourceRecord
+
+	host1 := acctest.RandString(10) + "ibm.com"
+	host2 := acctest.RandString(10) + "ibm.com"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckIBMDNSDomainDestroy,
+		Steps: []resource.TestStep{
+			{
```

Comment:
Is the test case working because I see ibm_dns_domain.test_dns_domain_records, ibm_dns_reverse_record.recordA and so on..We are trying to create a reverse record but the validation is checking for domain and record A type..

Reply:
We have used dns_domain record as there is no specific way to check for reverse_dns_record.
So even if we are provisioning Rdns ,we need to fetch details from Dns related helpers.

Reply:
Can you share me the result of this testcase . From the view I can see the testcase will fail because you are not creating any resources ibm_dns_domain.test_dns_domain_records or ibm_dns_reverse_record.recordA..

make testacc TEST=./ibm TESTARGS='-run=TestAccIBMDNSReverseRecord_Basic'

Reply:
I've run the testcase with Visual Code .
Output for the test case -

`Running tool: /usr/local/go/bin/go test -timeout 30s github.com/IBM-Cloud/terraform-provider-ibm/ibm -run ^(TestAccIBMDNSReverseRecord_Basic)$

ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm	(cached)
Success: Tests passed.`